examples/arc/app.js

```diff
@@ -1,11 +1,10 @@
 /* global window,document */
 import React, {Component} from 'react';
+import {fetch} from 'global/window';
```

Comment:
This requires installing `global` in every example, if not the examples won't build if you clear your root node_modules. 

And there is not much point of removing one dependency if we have to add another. So it might be better to just add a `/* global fetch */` comment to appease the linter.